<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CamSettingsDlg</class>
 <widget class="QDialog" name="CamSettingsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Camera Settings</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>311</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <spacer>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>131</width>
        <height>31</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>311</width>
     <height>94</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="verticalSpacing">
     <number>16</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_Exposure">
      <property name="text">
       <string>Exposure</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSlider" name="horizontalSlider_Exposure">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QSpinBox" name="spinBox_Exposure"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_Gain">
      <property name="text">
       <string>Gain</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSlider" name="horizontalSlider_Gain">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QSpinBox" name="spinBox_Gain"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_Level">
      <property name="text">
       <string>Level</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSlider" name="horizontalSlider_Level">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QSpinBox" name="spinBox_Level"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>CamSettingsDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_Exposure</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_Exposure</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_Gain</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_Gain</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_Level</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_Level</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
